contribs/github-bot/internal/requirements/reviewer.go

```diff
@@ -36,10 +36,21 @@ func deduplicateReviews(reviews []*github.PullRequestReview) []*github.PullReque
 				result = append(result, rev)
 				added[rev.User.GetLogin()] = len(result) - 1
 			}
+		case utils.ReviewStateDismissed:
+			// this state just dismisses any previous review, so remove previous
+			// entry for this user if it exists.
+			if ok {
+				result[idx] = nil
```

Comment:
is rev.User the user dismissing or the one being dismissed?

Reply:
I just tested on my test repo and I confirm that it is indeed the author of the original review.